docs/hsi-tests.d/org.fwupd.hsi.IntelCet.Enabled.json

```diff
@@ -11,14 +11,11 @@
     "not-supported": "CET not supported"
   },
   "success-results": {
-    "enabled": "CET feature enabled by the platform"
+    "supported": "CET feature support by the platform"
```

Comment:
I think we need both here to avoid regressions with insights

Reply:
fixed